scripts/train/yamls/pretrain/gpt2-small.yaml

```diff
@@ -73,8 +73,8 @@ algorithms:
     clipping_type: norm
     clipping_threshold: 1.0
 
-max_duration: 4800ba # ~ 2.5B tokens
-eval_interval: 500ba
+max_duration: 33696ba # ~ 17.55B tokens; 150 tokens / param
```

Comment:
"`17.55B tokens`" from the original paper?

Reply:
@samhavens did you by any chance make this change? I might have but I don't remember. In any case, I've lost track of why this change was made.

Reply:
I tried to make this training example more exciting by making it LLaMa ratios; we can revert this change though.

Reply:
I'm good either way. @vchiley any objections to keeping it spicy?

Reply:
its an example training yaml. doesn't really matter I'm good with it.